Zotero/Scenes/Detail/DetailCoordinator.swift

```diff
@@ -495,15 +503,14 @@ extension DetailCoordinator: DetailItemsCoordinatorDelegate {
             tags: tags,
             parentTitleData: parentTitleData,
             title: title,
-            saveCallback: saveCallback,
             navigationController: navigationController,
             controllers: controllers
         )
         coordinator.parentCoordinator = self
         childCoordinators.append(coordinator)
         coordinator.start(animated: false)
 
-        return navigationController
+        return (navigationController, coordinator.viewModel!)
```

Comment:
Is there a simple way to not force the `coordinator.viewModel` parameter? Although, since the coordinator will be started first, it should always be safe.

Reply:
I could also store the view model in `coordinator` and use that, but I didn't really want to create that dependency, it should always be fine here. But I'll at least add a comment for future reference that this has to be called after start.